src/app/modules/registration/components/edit-images/edit-images.component.html

```diff
@@ -19,6 +19,42 @@
       [rows]="2"
       placeholder="{{ pictureCommentPlaceholder() | translate }}"
     ></app-text-comment>
+    <ion-accordion-group>
+      <ion-accordion>
+        <ion-item slot="header" class="accordion-button">
+          <ion-label>{{ "REGISTRATION.SNOW.SNOW_PROFILE.STRAT_PROFILE.SHOW_MORE" | translate }}</ion-label>
+        </ion-item>
+        <div class="ion-padding" slot="content">
+          <ion-item>
+            <app-select
+              label="REGISTRATION.IMAGE_ASPECT"
+              [(selectedValue)]="attachment.Aspect"
+              [options]="aspectOptions()"
+            ></app-select>
+          </ion-item>
+          <ion-item>
+            <ion-input
+              [value]="attachment.Photographer"
+              (ionChange)="updateExistingAttachmentPhotographer(attachment, $event)"
+              [label]="'MY_PROFILE.PHOTOGRAPHER' | translate"
```

Comment:
Andre labels i appen vises upper case. Sikkert lurt å gå for det samme her. Du kan bruke en "uppercase" pipe tror jeg. Søk på `| uppercase` i prosjektet, så finner du andre steder den er brukt.